aidants_connect_web/tests/test_views/test_new_mandat.py

```diff
@@ -20,7 +20,7 @@
 class NewMandatTests(TestCase):
     def setUp(self):
         self.client = Client()
-        self.aidant_thierry = factories.UserFactory()
+        self.aidant_thierry = factories.AidantFactory()
```

Comment:
Est-ce que ça ne vaudrait pas le coup d'importer directement `AidantFactory` ?
```py
from aidants_connect_web.tests. factories import AidantFactory
```
Puis de faire : 
```suggestion
        self.aidant_thierry = AidantFactory()
```

Reply:
Pourquoi pas :) Je change ça !